test/integration/features/cmd-addons.feature

```diff
@@ -3,13 +3,41 @@ Feature: Addons command and its subcommands
 
   @minishift-only
   Scenario: Default add-ons are installed but not enabled by default
-     Note: default addons were installed during previous commands automatically.
+     Note: default add-ons were installed during previous commands automatically.
      When executing "minishift addons list"
      Then stdout should match "admin-user\s*: disabled\s*P\(0\)"
       And stdout should match "anyuid\s*: disabled\s*P\(0\)"
       And stdout should match "registry-route\s*: disabled\s*P\(0\)"
       And stdout should match "xpaas\s*: disabled\s*P\(0\)"
 
+@minishift-only
+  Scenario: Verbose listing of add-ons installed by default
+     When executing "minishift addons list --verbose"
+     Then stdout should match "Name\s*: admin-user"
+      And stdout should match "Description\s*: Create admin user and assign the cluster-admin role to it."
+      And stdout should match "Enabled\s*: false"
+      And stdout should match "Priority\s*: 0"
+      And stdout should match "Url\s*:"
+      And stdout should match "Openshift Version\s*:"
+      And stdout should match "Name\s*: anyuid"
+      And stdout should match "Description\s*: Changes the default security context constraints to allow pods to run as any user."
+      And stdout should match "Enabled\s*: false"
+      And stdout should match "Priority\s*: 0"
+      And stdout should match "Url\s*:"
```

Comment:
Each
```
      And stdout should match "Url\s*:"
```
will not match sequentially

Reply:
@gbraad I am planning to create an issue to track matching line by line sequentially where we can help regular expression then we can fix it.

Reply:
It makes sense to change this to

```
    Then stdout should contain
     """
Name: admin-user
Description: Create admin user and assign the cluster-admin role to it.
Enabled: false
Priority: 0
Url:
Openshift Version:
     """
```

or similar, with multiple "match" as it would make the Scenario more readable.

Reply:
At the moment the scenario is not pleasant to read due to the many statements.